qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -325,3 +334,63 @@ def iterations(self, value: List[int]) -> None:
             value: A new value for the powers.
         """
         self._iterations = value
+
+    @property
+    def circuit_results(self) -> Optional[Union[np.ndarray, Dict[str, int]]]:
+        """Return the circuit results. Can be a statevector or counts dictionary."""
+        return self._circuit_results
+
+    @circuit_results.setter
+    def circuit_results(self, value: Union[np.ndarray, Dict[str, int]]) -> None:
+        """Set the circuit results."""
+        self._circuit_results = value
+
+    @property
+    def top_measurement(self) -> str:
+        """Return the circuit results. Can be a statevector or counts dictionary."""
+        return self._top_measurement
+
+    @top_measurement.setter
+    def top_measurement(self, value: str) -> None:
+        """Set the circuit results."""
+        self._top_measurement = value
+
+    @property
+    def max_probability(self) -> float:
```

Comment:
Are any of these new values something that we would want to have in the general AmplitudeAmplifierResult, that this class extends, so they are commonly supported or are they specific to grover in which case they are fine here?

Reply:
Actually, the general `AmplitudeAmplifierResult` already had those properties. I will remove them from `GroverResult'.